src/snowflake/snowpark/file_operation.py

```diff
@@ -338,7 +339,9 @@ def get_stream(
         else:
             options = {"parallel": parallel}
             tmp_dir = tempfile.gettempdir()
-            src_file_name = stage_location.rsplit("/", maxsplit=1)[1]
+            src_file_name = unwrap_single_quote(stage_location).rsplit("/", maxsplit=1)[
+                1
+            ]
```

Comment:
does this change also needs to apply to `get` / `put` / `put_stream`? 

note to self: run stored-proc test before merging

Reply:
`get()` seems to work fine with just the `utils` change. Still working on testing the sproc path, will also look into the `put`/`put_stream` paths.

Reply:
one note: many `snow://` stage types don't allow write/`PUT`. It's still worth ensuring the client code works but that path may not work end to end

Reply:
Applied relevant changes to `put_stream`.